tests/scripts/test_pkgdev_mask.py

```diff
@@ -182,18 +182,53 @@ def test_existing_masks(self):
         assert self.profile.masks == frozenset([atom_cls('cat/masked'), atom_cls('=cat/pkg-0')])
 
     def test_last_rites(self):
-        with os_environ(EDITOR="sed -i '1s/$/mask comment/'"), \
-                patch('sys.argv', self.args + ['cat/pkg', '-r']), \
-                pytest.raises(SystemExit), \
-                chdir(pjoin(self.repo.path)):
-            self.script()
-
         removal_date = self.today + timedelta(days=30)
         today = self.today.strftime('%Y-%m-%d')
         removal = removal_date.strftime('%Y-%m-%d')
-        assert self.masks_path.read_text() == textwrap.dedent(f"""\
-            # First Last <first.last@email.com> ({today})
-            # mask comment
-            # Removal: {removal}
-            cat/pkg
-        """)
+        for rflag in ('-r', '--rites'):
+            with os_environ(EDITOR="sed -i '1s/$/mask comment/'"), \
+                    patch('sys.argv', self.args + ['cat/pkg', rflag]), \
```

Comment:
One could arguably extend this test a little further to test `--rites <N>` as well.

Reply:
I can do it in a follow-up PR :-) I considered pulling this specific commit into its own PR.